client/me/account-close/confirm-dialog.jsx

```diff
@@ -65,26 +65,37 @@ class AccountCloseConfirmDialog extends Component {
 	};
 
 	render() {
-		const { currentUsername, isVisible, translate } = this.props;
+		const { currentUsername, siteCount, isVisible, translate } = this.props;
 		const isDeleteButtonDisabled = currentUsername && this.state.inputValue !== currentUsername;
 
 		const alternativeOptions = [
-			{
-				englishText: 'Start a new site',
-				text: translate( 'Start a new site' ),
-				href: onboardingUrl() + '?ref=me-account-close',
-				supportLink: localizeUrl(
-					'https://wordpress.com/support/create-a-blog/#adding-a-new-site-or-blog-to-an-existing-account'
-				),
-				supportPostId: 3991,
-			},
-			{
-				englishText: "Change your site's address",
-				text: translate( "Change your site's address" ),
-				href: '/settings/general',
-				supportLink: localizeUrl( 'https://wordpress.com/support/changing-site-address/' ),
-				supportPostId: 11280,
-			},
+			...( siteCount > 0
+				? [
+						{
+							englishText: 'Start a new site',
```

Comment:
I think we should leave this option in if they have 0 sites.

Reply:
Sure, updated.